org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -115,6 +115,25 @@
                 "test -n \"$COMPAT_TARGET\" && cd \"rust-$COMPAT_TARGET\" && ./install.sh --prefix=/usr/lib/sdk/rust-stable --disable-ldconfig --verbose --components=rust-std-$COMPAT_TARGET,rust-analysis-$COMPAT_TARGET"
             ]
         },
+        {
+            "name": "rust-src",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rust-lang/rust/archive/1.48.0.tar.gz",
+                    "sha256": "d5a72a5a375f147f90dd980d88b2625ed84f4944cab25ecf353da8f00d1c25c5",
+                    "x-checker-data": {
+                        "type": "anitya",
```

Comment:
Consider using the new rust checker here. There is `rust-src` package, I guess it's effectively the same thing?

Reply:
Good point.
I've opened https://github.com/flathub/org.freedesktop.Sdk.Extension.rust-stable/pull/29 in favour of this one